### DISCUSSION TITLE
Unicode support?

### DISCUSSION THREAD

**Discussion by s25765** [mikeberger/borg_calendar#146] [Ideas]
When I entered some Chinese characters, they appeared as boxes.
Seems like Unicode is not yet supported by this program.
This is a great program already, but I think it could have wider use for people in other countries if it supports Unicode.
Thank you for your great work!

**Comment by mikeberger** [comment]
Change the Default font in the font options. The default is Arial, which doesn't seem to work.
Also set the Locale at the bottom of the appearance options and restart.
When I change to Sans Serif and set a Chinese locale, I get the image below.
A chinese translation was provided many years ago. A lot of new text has been added since then, so a lot is missing.

![cn](https://user-images.githubusercontent.com/6563665/140573933-d516b8e4-4ee9-44d0-a075-ca8c03fc60c2.JPG)

**Reply by s25765** [reply]
Yes! I followed your instructions and now it works beautifully! Thank you!!
